Document/0x05b-Basic-Security_Testing.md

```diff
@@ -752,16 +752,49 @@ Check the [Termux Wiki](https://wiki.termux.com/wiki/Remote_Access "Termux Remot
 
 #### Obtaining and Extracting Apps
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1238>
+There are several ways of extracting apk files from a device. You will need to decide which one is the easiest method depending if the app is public or private.
 
 ##### App Store
 
+One of the easiest option is to download the apk from websites that mirror public applications from the Google Play Store. However, keep in mind that these sites are not offical and there is no guarantee that the application hasn't been repackaged or contain malware. A few reputable websites that host apks and are not known for modifying apps and claim to MD5 checksum the apps are:
```

Comment:
> hasn't been repackaged or contain malware 

Sounds a bit weird (though might be technically correct). How about:

> hasn't been repackaged to contain malware

Reply:
Yeah, there were some cases where they repackaged the application to change the admob id so the revenue went to a different team so not exactly malware there.